src/signals/IncidentMap/components/IncidentMap/IncidentMap.tsx

```diff
@@ -66,6 +66,19 @@ export const IncidentMap = () => {
     resetMarkerIcons()
   }, [resetMarkerIcons])
 
+  const toggleFilter = useCallback(
+    (categoryName: Filter[], checked: boolean) => {
+      let updated = filters
+      categoryName.forEach((category) => {
+        if (category.filterActive != checked) {
+          updated = updateFilterCategory(category.name, updated)
+        }
+      })
+      setFilters(updated)
+    },
+    [filters]
+  )
+
```

Comment:
No need to move this function to IncidentMap. In the line of separation of concerns, this belongs to the FilterPanel component.

Reply:
true